include/slang-rhi.h

```diff
@@ -1295,7 +1296,7 @@ struct AccelerationStructureBuildInputInstances
     uint32_t instanceCount;
 };
 
-static const uint32_t kMaxAccelerationStructureMotionKeyCount = 2;
+inline constexpr uint32_t kMaxAccelerationStructureMotionKeyCount = 2;
```

Comment:
We should probably change over the other `static const` constants as well, but this can be a separate PR.